hbase-client/src/main/java/org/apache/hadoop/hbase/client/TableDescriptorBuilder.java

```diff
@@ -85,6 +85,15 @@ public class TableDescriptorBuilder {
   public static final String COMPACTION_ENABLED = "COMPACTION_ENABLED";
   private static final Bytes COMPACTION_ENABLED_KEY = new Bytes(Bytes.toBytes(COMPACTION_ENABLED));
 
+  /**
+   * Used by HBase Shell interface to access this metadata attribute which denotes if the table
```

Comment:
Are we planning to expose this via shell as another commit?

Reply:
Now, we can use shell to change the table description with SKIP_MERGING_COMPACTION.
alter 'tableName',CONFIGURATION => {'SKIP_MERGING_COMPACTION' => 'true'}